firefox-ios/Client/Frontend/Home/Pocket/PocketViewModel.swift

```diff
@@ -208,15 +208,19 @@ extension PocketViewModel: HomepageSectionHandler {
         recordSectionHasShown()
 
         if isStoryCell(index: indexPath.row) {
-            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: LegacyPocketStandardCell.cellIdentifier,
-                                                          for: indexPath) as! LegacyPocketStandardCell
+            guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: LegacyPocketStandardCell.cellIdentifier,
+                                                                for: indexPath) as? LegacyPocketStandardCell else {
+                fatalError("Failed to dequeue cell with identifier \(LegacyPocketStandardCell.cellIdentifier)")
```

Comment:
this is an improvement to what we had before while also mimicking similar to what we had. If we want to fail more gracefully, we can also return a generic `UICollectionViewCell` with a fatal log error.